### DISCUSSION TITLE
Increasing the size of the stat_list card

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2042] [Q&A] [answered]
Is it possible to increase the size of the `ui.stat_list()` card to fill up the entire space in the zone?

**Comment by mturoci** [comment]
Can you add an example? `stat_list` is just a bunch of rows IIRC so I am not sure what you mean by filling up the entire space. If there are not enough rows, there is not much we can do about it, but maybe I am missing something?

**Comment by n-srinidhi** [comment]
![image](https://github.com/h2oai/wave/assets/117831190/dd90d485-f368-441a-badc-041982a2ad07)

I have 5 records in this list, I just want it to increase in size a bit so as to fill up the entire zone, and not have the black space displayed after the last line, I have a graph alongside this `stat_list` card, so this is the size of the zone.

**Reply by mturoci** [reply, marked as answer]
I see what you are asking now. That's not possible as it would look odd. Stat_list rows are fixed in size by-design.

**Reply by n-srinidhi** [reply]
Alright. Thank you.
